google_drive/tests/integration/test_studio.py

```diff
@@ -0,0 +1,134 @@
+""" Runs tests for the studio views """
+
+from ddt import ddt, unpack, data
+from .base_test import GoogleCalendarBaseTest, GoogleDocumentBaseTest
+from studio_scenarios import calendar_scenarios, document_scenarios, image_scenarios
+
+DEFAULT_CALENDAR_SRC = (
+    'https://www.google.com/calendar/embed?'
+    'mode=Month&'
+    'src=edx.org_lom804qe3ttspplj1bgeu1l3ak@group.calendar.google.com&'
+    'showCalendars=0'
+)
+
+DEFAULT_DOCUMENT_SRC = (
+    'https://docs.google.com/presentation/d/1x2ZuzqHsMoh1epK8VsGAlanSo7r9z55ualwQlj-ofBQ/embed?'
+    'start=true&loop=true&delayms=10000'
+)
+
+TEST_IMAGE_SRC = 'https://docs.google.com/drawings/d/1lmmxboBM5c_0WCTjhAxBdkpqQb3T8VSwtuG0TRR1ODQ/pub?w=960&h=720'
+
+
+
+@ddt
+class GoogleCalendarStudioTest(GoogleCalendarBaseTest):
```

Comment:
@antoviaque Thanks for providing an example of studio testing. Here are integration tests which include studio views.